src/utils/constants.ts

```diff
@@ -8,8 +8,27 @@ export const CHECKOUT_SANDBOX =
 export const CHECKOUT_DEVELOPMENT =
   'https://dev.api.getsafepay.com/checkout/pay'
 
+export const SUBSCRIPTION_PRODUCTION =
+  'https://getsafepay.com/checkout/subscribe'
+export const SUBSCRIPTION_SANDBOX =
+  'https://sandbox.api.getsafepay.com/checkout/subscribe'
+export const SUBSCRIPTION_DEVELOPMENT =
+  'https://dev.api.getsafepay.com/checkout/subscribe'
+
```

Comment:
can we make this more DRY by defining a baseURL and then specific paths for checkout and subscription?

right now we are repeating the baseURLs quite frequently throughout this file